README.md

```diff
@@ -56,7 +56,7 @@ full access to AWS Simple Storage Service (S3)
 ```hcl
 module "role-s3-full-access" {
   source  = "mineiros-io/iam-role/aws"
-  version = "~> 0.6.0"
+  version = "~> 0.6.1"
```

Comment:
we don't need to specify a patch when using a range according to semver https://semver.org/

Reply:
Good point, you've said this before and I forgot.

Reply:
In that case I will close this PR since it's no-longer necessary.